index.js

```diff
@@ -26,44 +26,44 @@ function sniff(buffer, callback) {
         if (header.indexOf('\"type\":') !== -1) {
             var m = /"type":\s?"(.+?)"/.exec(header);
             if (!m) {
-              return callback(invalid('Unknown filetype'));
+                return callback(invalid('Unknown filetype'));
```

Comment:
cool, lets make it consistent with our [other repos](https://github.com/mapbox/preprocessorcerer/blob/master/.eslintrc).